fastlane/lib/fastlane/actions/upload_symbols_to_crashlytics.rb

```diff
@@ -94,7 +94,9 @@ def self.details
         [
           "This action allows you to upload symbolication files to Crashlytics.",
           "It's extra useful if you use it to download the latest dSYM files from Apple when you",
-          "use Bitcode"
+          "use Bitcode. This action will not fail the build if one of the uploads failed.",
+          "The reason for that is that sometimes some of dSYM files are invalid, and we don't want",
```

Comment:
I am wondering if it could be
* network issues while downloading
* file encoding issues ?

is there any information we can use (like md5s or any other checksum) that could help troubleshoot this ? I just find it suspicious that some files just don't work :)

We could compare the files downloaded by fastlane to those available online for example.